recipes/cpp-httplib/all/test_package/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ conan_basic_setup()
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
-set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Comment:
not sure if there's a huge difference between these two, but I am okay with either one

Reply:
the result is the same, but `set_target_properties` requires CMake 3+.